src/domain/usecases/GetSurveyAntibioticsBlacklistUseCase.ts

```diff
@@ -0,0 +1,11 @@
+import { FutureData } from "../../data/api-futures";
+import { Id } from "../entities/Ref";
+import { SurveyRepository } from "../repositories/SurveyRepository";
+
+export class GetSurveyAntibioticsBlacklistUseCase {
+    constructor(private surveyRepository: SurveyRepository) {}
+
+    public execute(surveyId: Id): FutureData<string[]> {
+        return this.surveyRepository.getSurveyAntibioticsBlacklist(surveyId);
```

Comment:
why is this use case required? The data is already fetched an exists in currentModule context

Reply:
not required, see comment below